plugins/builds/index.js

```diff
@@ -995,6 +995,21 @@ exports.register = (server, options, next) => {
                     let newBuild;
                     let parentBuildsForJoin = joinParentBuilds;
 
+                    // Get join information in context of join job
+                    const nextJobsForJoin = workflowParser.getNextJobs(parentWorkflowGraph, {
+                        trigger: fullCurrentJobName,
+                        chainPR: externalPipeline.chainPR
+                    });
+                    const joinObjForJoin = nextJobsForJoin.reduce((obj, jobName) => {
+                        obj[jobName] = workflowParser
+                            .getSrcForJoin(parentWorkflowGraph, { jobName });
+
+                        return obj;
+                    }, {});
+                    const joinListForJoin = joinObjForJoin[externalJobName];
+                    const joinListNamesForJoin = joinListForJoin ?
```

Comment:
if this is `undefined`, won't it break `getParentBuildStatus` https://github.com/screwdriver-cd/screwdriver/pull/1979/files#diff-7a514e440aa937beee722c7ac84dfb93R1075
Because `getParentBuildStatus` https://github.com/screwdriver-cd/screwdriver/blob/fc877d624044bda152fccef0c029404004a64a1d/plugins/builds/index.js#L542-L549 assumes it's an array